internal/cmd/object-storage/credentials/delete/delete.go

```diff
@@ -0,0 +1,113 @@
+package delete
```

Comment:
In general we should always say `credentials` in plural, to match the command name